robocorp-code/vscode-client/src/inspector/inspectorView.ts

```diff
@@ -0,0 +1,77 @@
+/**
+ * Interesting docs related to webviews:
+ * https://code.visualstudio.com/api/extension-guides/webview
+ */
+import path = require("path");
+import { readFileSync } from "fs";
+
+import * as vscode from "vscode";
+
+import { getExtensionRelativeFile, verifyFileExists } from "../files";
+import { logError } from "../channel";
+import { getSelectedRobot } from "../viewsCommon";
+import { LocatorsMap } from "./types";
+import { IMessage, IMessageType } from "./protocols";
+
+export async function showInspectorUI(context: vscode.ExtensionContext) {
+    const panel = vscode.window.createWebviewPanel(
```

Comment:
As a note, with this API it can only be created in the center (customizable would be `Active` or `Beside` columns) -- with this approach `Beside` may be better?

To put it in some other pre-defined place (such as `panel` which is in the bottom) it needs to be added to the `package.json` (see `codegen/views.py`) and then a `vscode.WebviewViewProvider` needs to be used.

If that's what we want you can search for `robocorp.python.view.output` and use that as a base (that's the `ROBO TASKS OUTPUT`).

Reply:
p.s.: No need to do this now, just wanted to point that out.

Reply:
Took a note on this. 
I haven't yet decided where this should view should be located. I'll try some other things out and see how they look. 
For now `Beside` seems to be the most appropriate. :)